pkg/metrics/metrics.jsx

```diff
@@ -229,6 +232,7 @@ class CurrentMetrics extends React.Component {
             netInterfacesTx: [],
             topServicesCPU: [], // [ { name, percent } ]
             topServicesMemory: [], // [ { name, bytes } ]
+            podNameMapping: {}, // { containerid -> name }
```

Comment:
I admittedly liked this better as `cgroup`, but as it's now a two-level dict, the comment is still wrong. It's "uid → containerid → name", isn't it?

Reply:
It is still wrong :|, it is indeed uid => containerid => name